central/debug/service/service.go

```diff
@@ -132,6 +133,14 @@ type serviceImpl struct {
 	notifierDataStore    notifierDS.DataStore
 }
 
+func (s *serviceImpl) PrivateDiagnosticsHandler() http.HandlerFunc {
```

Comment:
please add a comment what `private` means like this is meant to not be exposed and accessed only with `kubectl`